chapters/equations.tex

```diff
@@ -629,10 +628,10 @@ \section{Events and Synchronization}\doublelabel{events-and-synchronization}
 
 Relations are taken literally also during continuous integration, if the
 relation or the expression in which the relation is present, are the
-argument of the \lstinline!noEvent(..)! function. The \lstinline!smooth(p,x)! operator also
+argument of \lstinline!noEvent!. \lstinline!smooth! also
```

Comment:
I don't like starting a sentence with a non-capital letter, would it be possible to rewrite the sentence starting with smooth?

Reply:
I guess we can consider this a _won't fix_ as of the last phone meeting?

Reply:
Yes.